.travis.yml

```diff
@@ -1,38 +1,40 @@
 language: node_js
 node_js:
-  - "4"
+  - "6"
 
 sudo: false
 
 cache:
   directories:
-    - node_modules
+    - $HOME/.npm
+    - $HOME/.cache # includes bowers cache
 
 env:
-  - EMBER_TRY_SCENARIO=ember-1.13
+  # we recommend testing LTS's and latest stable release (bonus points to beta/canary)
   - EMBER_TRY_SCENARIO=ember-lts-2.4
+  - EMBER_TRY_SCENARIO=ember-lts-2.8
   - EMBER_TRY_SCENARIO=ember-release
-  - EMBER_TRY_SCENARIO=ember-beta
-  - EMBER_TRY_SCENARIO=ember-canary
```

Comment:
I think the failures for these versions are related to using phantomjs in CI, and chrome should be used instead? Or they're real failures that I should fix

I'll do that myself after merging this, so you won't have to. I think `testem.js` needs to be changed to something like [this](https://github.com/stefanpenner/ember-moment/blob/master/testem.js) to use chrome in CI

Reply:
ah I see, I'm sure this is not the first addon to use unix time integers as params, so I'm sure there's a solution somewhere